src/modules/links/renderer/node/Links.jsx

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
remainder we need to add a specific eslint config for modules' renderer part.

Reply:
same for jest (this is not going to be fun)

Reply:
Can you please open an issue for that so we can postpone it?

Reply:
https://github.com/developer-vault/developer-vault/issues/65